### PR TITLE
bugfix/11758-stack-labels-styled-mode

### LINKED ISSUE #11758: StyledMode + stackLabels error
yAxis.stackLabels combined with styledMode produce an error:
`30Error: <rect> attribute width: Unexpected end of attribute. Expected length, "".`

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/Lta2xdor

#### Product version
7.1.3

#### Affected browser(s)
All (chart is not displayed in FF at all)
Related to #10855

### PR DESCRIPTION
**Pull request by pawelfus**
Fixed #11758, `stackLabels` in styled mode used to throw errors.

### CODE REVIEW

File: ts/parts/SvgRenderer.ts [hunk 1]
```diff
@@ -6087,6 +6087,9 @@ extend(SVGRenderer.prototype, /** @lends Highcharts.SVGRenderer.prototype */ {
                     anchorY: anchorY
                 });
             }
+
+            updateBoxSize();
+            updateTextPadding();
```
Comment (TorsteinHonsi) [inline ts/parts/SvgRenderer.ts]: The `updateBoxSize` and `updateTextPadding` functions are called from `wrapper.textSetter`, which should be invoked from line 6079. Why do we need to call them again here?

Comment (pawelfus) [reply ts/parts/SvgRenderer.ts]: Oh, right, I missed that! I thought updating box/padding was not called on add.

**Comment by swrobel** [comment]
It seems this just missed the cut for 7.2.1. Any chance of cutting another release soon?

**Comment by pawelfus** [comment]
Hi @swrobel 

v7.2.1 was released ~2 weeks ago, next release should be in December

**Pull request merged**